apps/reference/_api/usage.mdx

```diff
@@ -430,6 +430,7 @@ GET https://api.supabase.com/v1/projects/{ref}/functions
         
         </span>
```

Comment:
we should move this file to a `generated` folder at some point (will make it easier to .gitignore it when we're ready)